packages/react-native-bpk-appearance/package.json

```diff
@@ -16,7 +16,7 @@
   "dependencies": {
     "bpk-tokens": "^30.1.0",
     "prop-types": "^15.5.8",
-    "react-native-dark-mode": "^0.2.1-rc.0"
+    "react-native-appearance": "github:tiagoengel/react-native-appearance"
```

Comment:
Why a fork? Just out of interest. Also, if we're gonna officially use a fork it should be under the `skyscanner` org.

Reply:
It's a fork because the official one doesn't work correctly on android unless you add a bit of code the the native app, which to me seems wrong and I'd like to avoid adding this to our app. I have this PR to change it so it would only be a fork until that is released, or if they refuse merging this for some reason we will need to add this extra code to the app.
https://github.com/expo/react-native-appearance/pull/35

Reply:
Ah ok cool. If, for some reason, we end up maintaining a fork for a long time, we should move it to Skyscanner. But this is all good for now.

Reply:
I'll move the fork to `Skyscanner` org :)

Reply:
Might be worth adding an explanation to the README too so that it doesn't get "cleaned up"